src/Components/ContributorsGrid/ContributorsGrid.jsx

```diff
@@ -9,11 +9,14 @@ const getDifferentColors = () => {
 };
 
 const ContributorsGrid = ({ contributors }) => {
+
+  const randomContributors = contributors.sort(() => Math.random - 0.5 )  
```

Comment:
Adjust the `sort()` function with a comparison function that returns positive, negative, or zero value to order elements.

Reply:
You are very close to making it work, and you got it!

Reply:
Sorry, I just realized I forgot to invoke the function which I assume why it isn't working. I wrote it out on  main first but deleted the commit and used a new branch and forgot to type it out correctly. 
Wouldn't changing it to 
const randomContributors = contributors.sort(() => Math.random() - 0.5 )  
suffice? I had that before when I was testing on main branch and it worked with 5 refreshes, each giving me a random contributor order every time

Reply:
Sorry, I'm just a bit confused at the need of a comparison function since we want it to be random and the inclusion of a comparison function would make it based on some type of criteria instead of holistically randomizing it. Sorry if this is a stupid question.

Reply:
> Sorry, I just realized I forgot to invoke the function which I assume why it isn't working. I wrote it out on main first but deleted the commit and used a new branch and forgot to type it out correctly. Wouldn't changing it to const randomContributors = contributors.sort(() => Math.random() - 0.5 ) suffice? I had that before when I was testing on main branch and it worked with 5 refreshes, each giving me a random contributor order every time

All good! When working on multiple branches and contributing to different projects, it can be easy to lose track of the latest updates. I've made this mistake before, so it's great that you were able to get it to work by using:
```
const randomContributors = contributors.sort(() => Math.random() - 0.5 )
```
It now correctly randomizes the contributors when viewing the preview deployment👏🏻🎉 Great work @Lozlink! And you were correct in missing invoking the function which is why it wasn't working or being utilized.

Reply:
> Sorry, I'm just a bit confused at the need of a comparison function since we want it to be random and the inclusion of a comparison function would make it based on some type of criteria instead of holistically randomizing it. Sorry if this is a stupid question.

Hello @Lozlink, thank you for your question. It's not a stupid question at all. I appreciate you asking because it makes me think deeper about using a comparison function for randomness. When we want to randomize the order of elements in an array, we can use the sort() method for simplicity.

Usually, the sort() function is utilized to organize elements based on certain criteria. However, in this situation, we want to incorporate randomness. To accomplish this, we can use a comparison function such as () => Math.random() - 0.5. This function produces a random number between -0.5 and 0.5 for every pair of elements being compared. If the random number is positive, the elements are interchanged; if it's negative, they remain in their original order. This process is repeated for all pairs of elements, resulting in a randomized order.  This can lead to a discussion of randomness and the Fisher-Yates algorithm for eliminating biased distribution as a rabbit randomness hole to go down😅.


Additional source to ponder come from this medium article on the fairness of shuffling for the randomness of a list - [https://itnext.io/stop-shuffling-with-sort-math-random-0-5-d5e6293c1f0c](https://itnext.io/stop-shuffling-with-sort-math-random-0-5-d5e6293c1f0c)
YoutTube video - [https://youtu.be/rlYincT2Qzc?si=6D_A_2adJsjTqN1k&t=312](https://youtu.be/rlYincT2Qzc?si=6D_A_2adJsjTqN1k&t=312)
Additional article from free to code on Math.random -  
[https://forum.freecodecamp.org/t/how-does-math-random-work-to-sort-an-array/151540](https://forum.freecodecamp.org/t/how-does-math-random-work-to-sort-an-array/151540)

Additionally you can check out this open source project [https://github.com/kubowania/memory-game](https://github.com/kubowania/memory-game) and see in the [App.js](https://github.com/kubowania/memory-game/blob/master/app.js) on line 54 the Comparison function in action to assist with creating a random shuffle to the functioning of the game. 

Hope this helps clarify and like so many topics in the world of functions and methods in opens up more doors to explore and learn.